src/components/Moteoversikt.tsx

```diff
@@ -27,10 +43,20 @@ const texts = {
 };
 
 const Moteoversikt = (): ReactElement => {
+  const { aktivEnhet } = useAktivEnhet();
   const [responsFilter, setResponsFilter] = useState<MoteRespons | "alle">(
     "alle"
   );
   const dialogmoterQuery = useMineDialogmoterQuery();
+  const veiledereFromEnhet = useGetVeiledere(aktivEnhet || "").data || [];
+  const [veilederIdent, setVeilederIdent] = useState<string>();
+  const { dialogmoterUuids, modifyDialogmoterUuids, isSelected } =
+    useDialogmoterUuids();
+  const tildelDialogmoter = useTildelDialogmoter(veilederIdent);
+  const [antallTildelteMoter, setAntallTildelteMoter] = useState<number>(0);
```

Comment:
Ville droppet `antallTildelteMoter` og `tildeltVeileder`-state som må synces med `veilederIdent` og `dialogmoterUuids`. Bruk heller sistnevnte direkte basert på `tildelDialogmoter.isSuccess` for å vise Alert-en under.